metadoc-js/src/main/scala/metadoc/MetadocApp.scala

```diff
@@ -125,7 +129,30 @@ object MetadocApp {
     )
   }
 
-  def openEditor(editorService: MetadocEditorService)(
+  def registerWorkspaceFiles(
+      editor: IStandaloneCodeEditor,
+      workspace: Workspace
+  ): Unit = {
+    workspace.filenames.foreach { file =>
+      editor.addAction(new IActionDescriptor {
```

Comment:
wdyt @jonas is this too hacky? 😅

Reply:
I like it. Creative given the limitations we currently have. But it would be good to clean up the PR a bit.